src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -364,8 +366,9 @@ public boolean dispatchTouchEvent(MotionEvent e)
 
 		if (!_isHitTestVisible || !_isEnabled)
```

Comment:
should visibility be checked here instead ?

Reply:
> Normally the visibility check would be handled by super.dispatch..., but this is the code path where we don't call it (because it doesn't account for the transform).

so it is ok as it is now.